src/libclipper/include/clipper/task_executor.hpp

```diff
@@ -24,6 +24,7 @@
 namespace clipper {
 
 const std::string LOGGING_TAG_TASK_EXECUTOR = "TASKEXECUTOR";
+constexpr size_t CACHE_SIZE_ELEMENTS = 2048;
```

Comment:
Now that you changed the cache size to be measured in bytes, should this variable be deleted?